src/window/window_wrapper.rs

```diff
@@ -410,7 +410,17 @@ impl WinitWindowWrapper {
         }
         skia_renderer.swap_buffers();
         if self.ui_state == UIState::FirstFrame {
-            skia_renderer.window().set_visible(true);
+            let window = skia_renderer.window();
+            let handle = window.window_handle().unwrap().as_raw();
```

Comment:
Please let me know if unwrapping the window handle is ok at this point, I guess it must be?